lib/pipeline.js

```diff
@@ -209,7 +209,12 @@ lunr.Pipeline.prototype.run = function (tokens) {
 
       if (result === void 0 || result === '') return memo
 
-      return memo.concat(result)
+      if (result instanceof Array) 
```

Comment:
The build is failing because this violates the current formatting lints in the project. Use braces for the if, e.g.

```javascript
if (...) {
  ...
} else {
  ...
}
```